File: 01.Java-Basic/06.ConditionalStatementsAdvancedExercise/src/Cinema.java
package ConditionalStatementsAdvancedExercise;

import java.util.Scanner;

public class Cinema {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String type = scanner.nextLine();
        int rows = scanner.nextInt();
        int cols = scanner.nextInt();

        double income = 0.0;
        if (type.equals("Premiere")) {
            income = rows * cols * 12.0;
        } else if (type.equals("Normal")) {
            income = rows * cols * 7.5;
        } else if (type.equals("Discount")) {
            income = rows * cols * 5;
        }

        System.out.printf("%.2f", income);
    }
}

File: JavaScript/01.JavaScriptBasic/06Nested-Conditional-Statements-Еxercise/src/Cinema.js
function income(input) {
    let income = 0;

    let type = input[0];
    let rows = Number(input[1]);
    let cols = Number(input[2]);

    let places = rows * cols;
    if (type === "Premiere") {
        income = places * 12.00;
    } else if (type === "Normal") {
        income = places * 7.50;
    } else if  (type === "Discount") {
        income = places * 5.00;
    }

    console.log(`${income.toFixed(2)} leva`);
}

income(["Premiere","10","12"]);